examples/single-project/main.tf

```diff
@@ -99,6 +99,11 @@ module "cloud_scanning" {
 #######################
 #      BENCHMARKS     #
 #######################
-module "cloud_bench" {
-  source = "../../modules/services/cloud-bench"
+module "config" {
+  source   = "../../modules/services/cloud-bench/config"
+}
+
+module "task" {
+  source = "../../modules/services/cloud-bench/task"
+  depends_on = [module.config]
```

Comment:
Instead of having the examples use both submodules, could we have the example create just a single `cloud_bench` module, and update the `main.tf` in the `cloud_bench` module to create these two submodules?

Reply:
Done!